doc/rst/developer/bestPractices/StandardModuleStyle.rst

```diff
@@ -130,7 +135,56 @@ Accessor methods will avoid using "get" in their name.  E.g., instead of
 
 Methods that are not accessors are still allowed to use "get" in their name.
 
+Formals
++++++++
+
+Formal names should be camelCase.  Descriptive names are recommended, within
```

Comment:
I would think that if variables and fields can be camelCase or PascalCase, that formal names should be able to be as well.  E.g., if you subscribe to a "[distributed|major] arrays should be PascalCase" worldview like mine and name your matrices `M` or `A`, it'd be reasonable to make a formal argument accepting such a matrix `X` rather than requiring it to be `x`.

Reply:
Oh that's where that came from.  I'm happy to restore the option for either, sorry about that!

Reply:
Opened #25377

Reply:
Thanks!